tests/test_format.py

```diff
@@ -132,6 +133,7 @@ def test_python_2_hint() -> None:
     exc_info.match(black.parsing.PY2_HINT)
 
 
+@pytest.mark.filterwarnings("ignore:invalid escape sequence.*:DeprecationWarning")
```

Comment:
this is caused by this example test case:

https://github.com/psf/black/blob/e026701b9a894ef19ad59eb6e394e5f3f7c18fdd/tests/data/miscellaneous/docstring_no_string_normalization.py#L107

at some point a future python version will cause this test to fail